src/codegen/c.zig

```diff
@@ -412,7 +412,19 @@ pub const DeclGen = struct {
         const int_info = @typeInfo(@TypeOf(int_val)).Int;
         const is_signed = int_info.signedness == .signed;
         const is_neg = int_val < 0;
-        comptime assert(int_info.bits > 64 and int_info.bits <= 128);
```

Comment:
why is this comptime assert deleted?

Reply:
This was wrongly deleted I reverted it, thanks.